src/utils/SharedUtils.js

```diff
@@ -0,0 +1,163 @@
+export function serialize(obj, prefix) {
```

Comment:
Ok, I'll undo that change then and go back to using jQuery for the things it makes sense to.